scripts/datasets/question_answering/prepare_squad.py

```diff
@@ -58,14 +59,18 @@ def main(args):
     download(dev_url, path=os.path.join(args.cache_path, dev_file_name))
     if not os.path.exists(args.save_path):
         os.makedirs(args.save_path)
-    if not os.path.exists(os.path.join(args.save_path, train_file_name))\
+    if not os.path.exists(os.path.join(args.save_path, train_file_name)) \
             or (args.overwrite and args.save_path != args.cache_path):
-        os.symlink(os.path.join(args.cache_path, train_file_name),
-                   os.path.join(args.save_path, train_file_name))
-    if not os.path.exists(os.path.join(args.save_path, dev_file_name))\
+        shutil.copyfile(os.path.join(args.cache_path, train_file_name),
+                        os.path.join(args.save_path, train_file_name))
+    else:
+        print(f'Found {os.path.join(args.save_path, train_file_name)}...skip')
+    if not os.path.exists(os.path.join(args.save_path, dev_file_name)) \
             or (args.overwrite and args.save_path != args.cache_path):
-        os.symlink(os.path.join(args.cache_path, dev_file_name),
-                   os.path.join(args.save_path, dev_file_name))
+        shutil.copyfile(os.path.join(args.cache_path, dev_file_name),
```

Comment:
Compare to ` os.symlink`, `shutil.copyfile` will produce an identical copy of the data. Do we need to consider the impact of this?

Reply:
The major reason is that symlink will have some issues when copied to the docker volume. Basically, you cannot copy the symbolic link. Thus, I choose to use copyfile directly.

Reply:
You can use `os.link` for hardlink to avoid copy